pkg/util/env.go

```diff
@@ -138,3 +139,9 @@ func GetElasticsearchMasterNodeReplicas() int32 {
 	}
 	return 3
 }
+
+// GetAccessControlAllowOrigins returns the additional allowed origins for API requests - these
+// will be added to the Access-Control-Allow-Origins response header of the API
+func GetAccessControlAllowOrigins() string {
```

Comment:
why an env var?

Reply:
We need this functionality of overriding the allowed origin so that we can do local testing - I was debating between an env var and yet another command line flag. Are you suggesting that we add another command line flag rather than an env var?